07-knitr-R.Rmd

```diff
@@ -1,8 +1,9 @@
 ---
-layout: page
```

Comment:
Please revert the YAML back to the original. We do not use the rmarkdown package to render these lessons. The YAML header is standard across the Markdown files for all the SWC lessons.